content/en/news/security/istio-security-2024-002/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: ISTIO-SECURITY-2024-002
+subtitle: Security Bulletin
+description: CVEs reported by Envoy.
+cves: [CVE-2024-27919, CVE-2024-30255, CVE-2023-45288]
+cvss: "7.5"
+vector: "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
```

Comment:
Vector yields a link with `vector: invalid CVSS metrics`

Reply:
Interesting, it gets pulled directly from the CVE and NVD parses it on the CVE Detail Page without issue. https://nvd.nist.gov/vuln/detail/CVE-2024-27919 -- wonder if that's a bug in their parser

Reply:
Actually I found the issue... I'll fix in a followup PR